### DISCUSSION TITLE
`{build.system.path}`

### DISCUSSION THREAD

**Discussion by JAndrassy** [earlephilhower/arduino-pico#1573] [General]
now when `system` folder is used, the `{runtime.platform.path}/system` path in platform.txt can be replaced with `{build.system.path}`

from https://arduino.github.io/arduino-cli/0.33/platform-specification/ :
>Moreover the following properties are automatically generated:
> *    {build.core.path}: The path to the selected board's core folder (inside the [core platform](https://arduino.github.io/arduino-cli/0.33/platform-specification/#platform-terminology), for example hardware/arduino/avr/core/arduino)
> *    {build.system.path}: The path to the [core platform](https://arduino.github.io/arduino-cli/0.33/platform-specification/#platform-terminology)'s system folder if available (for example hardware/arduino/sam/system)
> *    {build.variant.path}: The path to the selected board variant folder (inside the [variant platform](https://arduino.github.io/arduino-cli/0.33/platform-specification/#platform-terminology), for example hardware/arduino/avr/variants/micro)

**Comment by earlephilhower** [comment]
I believe on a release all the instances of `{r.p.p}/system` are removed:
https://github.com/earlephilhower/arduino-pico/blob/0f437e4db29e61580cbd76fb7cd4115bcabfde32/package/build_boards_manager_package.sh#L106-L128

For the `git` version, if you want to use the non-IDE installed versions of the tools I am pretty sure we still need the path as-is.

Are you seeing any issues with the current setup?
